README.md

```diff
@@ -16,19 +16,33 @@ You now no longer register your DID resolvers using the global `registerMethod()
 
 ## Configure `Resolver` object
 
-You are now required to preconfigure a resolver during instantiation:
+You are now required to preconfigure a resolver during instantiation. The `Resolver` constructor expects a registry of methods mapped to a resolver function. For example: 
+```
+{ 
+  ethr: resolve,
+  web: resolve
+}
+```
 
+Each method resolver should expose a function called `getResolver` which will return an object containing one of these key/value pairs. Then you can flatten them into one object to pass into the `Resolver` constructor. 
```

Comment:
This stands in contrast to the instructions on how to develop a DID method below. I guess this is derived from the fact that `ethr-did-resolver` and `web-did-resolver` implemented it this way.

Reply:
Good point, I've updated that section. Thanks for the feedback and let me know if anything else looks off.